File: DocumentProject.WebAPI/DocumentProject.WebAPI/Data/Application.cs
using DocumentProject.WebAPI.Data.Abstract;

namespace DocumentProject.WebAPI.Data
{
    public class Application: Entity
    {
        public string Number { get; set; }
        public string Name { get; set; }
        public string Status { get; set; }
        public DateTime? SignatureDate { get; set; }
        public Guid MemberId { get; set; }
        public Member Member { get; set; }
        public Guid OrganizationId { get; set; }
        public Organization Organization { get; set; }
    }
}

File: DocumentProject.WebClient/src/app/_models/application.ts
import { BaseModel } from "./abstract/base-model";
import { Member } from "./member";
import { Organization } from "./organization";

export class Application extends BaseModel { 
  number: string | undefined;
  name: string | undefined;
  status: string | undefined | null = null;
  type: string | undefined;
  signatureDate: Date | null = null;
  text: string | undefined | null = null;
  memberId: string | undefined;
  member: Member | undefined;
  organizationId: string | undefined;
  organization: Organization | null = null;

  document: string | undefined | null = null;
}